### PR TITLE
feat(vscode): add CodeLens to support editing encrypted secrets

### PR DESCRIPTION
**Pull request by tecton**
With the change of encryption, it also changes the UX of managing secrets. It should be used in rare cases:
In VS Code, CodeLens is used to provide a better experience.

VS Code Preview (gif):
vsc-crypto

### CODE REVIEW

File: packages/vscode-extension/src/telemetry/extTelemetryEvents.ts [hunk 1]
```diff
@@ -79,6 +79,8 @@ export enum TelemetryEvent {
   DebugNpmInstall = "debug-npm-install",
 
   Survey = "survey",
+
+  EditSecret = "edit-secret",
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/telemetry/extTelemetryEvents.ts:83]: Do you need another event to indicate whether edit secret is successful or not? You can see from the event file that the common pattern is event-name-start for start and event-name with property success(yes/no) to indicate the result

Comment (tecton) [reply packages/vscode-extension/src/telemetry/extTelemetryEvents.ts:84]: Thanks, updated.

File: packages/vscode-extension/src/handlers.ts [hunk 2]
```diff
@@ -791,6 +791,44 @@ export async function cmpAccountsHandler() {
   quickPick.show();
 }
 
+export async function decryptSecret(cipher: string, selection: vscode.Range): Promise<void> {
+  ExtTelemetry.sendTelemetryEvent(TelemetryEvent.EditSecretStart, {
+    [TelemetryProperty.TriggerFrom]: TelemetryTiggerFrom.Other,
+  });
+  const editor = vscode.window.activeTextEditor;
+  if (!editor) {
+    ExtTelemetry.sendTelemetryEvent(TelemetryEvent.EditSecret, {
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/handlers.ts:800]: Use sendTelemetryErrorEvent() instead

Comment (tecton) [reply packages/vscode-extension/src/handlers.ts:800]: Updated.

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by 1openwindow — APPROVED** [review]

**Pull request merged**